src/admin-views/settings/tickets-commerce/gateways.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * TEmplate to display a list of gateway options.
```

Comment:
This whole file should make better use of template variables.

Example `Tribe__Tickets__Main::instance()->plugin_url`